### PR TITLE
update no-return-assign

### PR DESCRIPTION
**Pull request by xiaokeqi**

### CODE REVIEW

**Comment by xkf521** [comment]
update docs/rules/no-return-assign.md			
 	update no-script-url.md,no-self-compare.md,no-sequences.md

File: docs/rules/no-sequences.md [hunk 1]
```diff
@@ -30,11 +32,11 @@ This rule forbids the use of the comma operator, with the following exceptions:
 
 * In the initialization or update portions of a `for` statement.
 
-在初始化或者`for`语句更新选项时。
+在初始化或者更新部分`for`语句时。
 
 * If the expression sequence is explicitly wrapped in parentheses.
 
-如果表达式顺序被显示的括号包裹。
+如果表达式序列被明确包裹在括号中。
```
Comment (molee1905) [inline docs/rules/no-sequences.md:39]: 括在括号中

File: docs/rules/no-return-assign.md [hunk 2]
```diff
@@ -20,7 +22,7 @@ function doSomething() {
 
 It is difficult to tell the intent of the `return` statement here. It's possible that the function is meant to return the result of `bar + 2`, but then why is it assigning to `foo`? It's also possible that the intent was to use a comparison operator such as `==` and that this code is an error.
 
-很难表述这里的`return`语句的意图。这是可能的,函数是为了返回`bar + 2`的结果,但是为什么赋值`foo`呢?也有可能目的是使用比较运算符比如`==`,如果是这样的话代码是错误的。
+从例子中，我们很难表述`return`语句的意图。可能意图一：函数是为了返回`bar + 2`,但是如果是这样的话，为什么赋值给`foo`呢?可能意图二：使用比较运算符比如`==`,如果是这样的话代码是错误的。
```
Comment (molee1905) [inline docs/rules/no-return-assign.md:25]: 待调整

File: docs/rules/no-return-assign.md [hunk 3]
```diff
@@ -31,7 +33,7 @@ Because of this ambiguity, it's considered a best practice to not use assignment
 
 This rule aims to eliminate assignments from `return` statements. As such, it will warn whenever an assignment is found as part of `return`.
 
-此规则目的在于从`return`语句中移除赋值。因此，每当在`return`中发现赋值它会给出警告。
+此规则目的在于从`return`语句中移除赋值。因此，每当在`return`中发现赋值ESLint会给出警告。
```
Comment (molee1905) [inline docs/rules/no-return-assign.md:37]: 该规则会发出警告

**Pull request merged**